demisto_sdk/commands/init/initiator.py

```diff
@@ -180,8 +257,13 @@ def content_item_to_package_format(self, content_item_dir: str, del_unified: boo
                 content_item_file_path = child_file
                 file_type = find_type(content_item_file_path)
                 file_type = file_type.value if file_type else file_type
-                extractor = Extractor(input=content_item_file_path, file_type=file_type, output=content_item_dir)
-                extractor.extract_to_package_format()
+                try:
+                    extractor = Extractor(input=content_item_file_path, file_type=file_type, output=content_item_dir)
+                    extractor.extract_to_package_format()
+                except Exception as e:
+                    err_msg = f'Error occurred while trying to split the unified YAML "{content_item_file_path}" ' \
+                              f'into its component parts.\nError: "{e}"'
+                    self.contrib_conversion_errs.append(err_msg)
```

Comment:
why are we collecting errors only from the unify ?

Reply:
That's the step where all the work is really happening.